extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -108,6 +128,80 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     }
     this.subscriptionService_.getDialog().close();
   }
+
+  /**
+   * @return {!Element}
+   * @private
+   * TODO(#37285): add "next story" button to the banner
+   */
+  renderSubscriptionsDialogTemplate_() {
+    return (
+      <div subscriptions-dialog subscriptions-display="NOT granted">
+        <div class="i-amphtml-story-subscriptions-dialog-banner"></div>
+        <div class="i-amphtml-story-subscriptions-dialog-content">
+          <div class="i-amphtml-story-subscriptions-price">
+            {this.element.getAttribute('price')}
+          </div>
+          {!!this.element.getAttribute('title') && (
+            <div class="i-amphtml-story-subscriptions-title">
+              {this.element.getAttribute('title')}
+            </div>
+          )}
+          {!!this.element.getAttribute('subtitle-first') && (
+            <div class="i-amphtml-story-subscriptions-subtitle-first">
```

Comment:
`div` elements are not visible in the a11y tree. This content will not be read.
This and any other element with text content needs to be in a text element 
`span` can be used here. It will need to be set to `display:block` since span is an inline element.

Please make this change here and elsewhere in the document.

Reply:
It looks like `subtitle-first` is required. We might not need to do this check here in that case.

Reply:
Exactly! This mock just got changed last night to reflect that since I am confused about if there's a case we just showing price.

Reply:
A handy Figma trick: clicking the "down arrow" next to the file name will display a "Show version history" button. It's a great way to double check if a change happened.

It looks like it's required since it's included in the most minimal state.
<img width="700" alt="Screen Shot 2022-04-14 at 11 17 05 AM" src="https://user-images.githubusercontent.com/3860311/163431329-a8fdb0e7-dbb9-44a6-860e-ead7981a65ea.png">